official/projects/pixel/README.md

```diff
@@ -0,0 +1,33 @@
+# Language Modelling with Pixels (PIXEL)
+
+TF2 implementation of [PIXEL](https://arxiv.org/abs/2207.06991).
+
+### Setup
+The current setup requires a numpyfied pytorch pixel model and preprocessed data.
+We have converted such a pre-trained model (along with sst2 data) and uploaded it to
```

Comment:
This gs is probably private. I think we should move them else where. Can you remove this for now and send us an email including the paths?